include/openspace/events/event.h

```diff
@@ -256,6 +258,39 @@ struct EventTimeOfInterestReached : public Event {
 };
 
 
+/**
+ * This event is created when a mission is added 
+ */
+struct EventMissionAdded : public Event {
+    static constexpr Type Type = Event::Type::MissionAdded;
+
+    /**
+     * Creates an instance of an EventMissionAdded event.
+     * 
+     * \param missionName The name of the mission added
+     */
+    EventMissionAdded(std::string_view name);
+
+    const tstring name;
+};
+
+/**
+ * This event is created when a mission is removed
+ */
+struct EventMissionRemoved : public Event {
+    static constexpr Type Type = Event::Type::MissionRemoved;
+
+    /**
+     * Creates an instance of an EventMissionRemoved event.
+     *
+     * \param missionName The name of the mission removed
```

Comment:
```suggestion
     * \param name The name of the mission removed
```

Reply:
For this all above in this event. Is it actually the `name` or is it the `identifier`?

Reply:
It seems like the identifier. lol

I think @engbergandreas talked about this recently?